umbra/umbra_64.c

```diff
@@ -1369,3 +1369,9 @@ umbra_handle_fault(void *drcontext, byte *target, dr_mcontext_t *raw_mc,
     }
     return false;
 }
+
+drmf_status_t
+umbra_clear_redundant_blocks(umbra_map_t *map) {
```

Comment:
style: { on own line

Reply:
Ah sorry about this. I have another coding style for my other projects and keep on missing such issues. When I applied clang-format on Umbra code, it modified the entire file - hence me avoiding formatting.

Reply:
I have #2107 on my list.

Reply:
Done.